compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/text/TextFieldKeyInput.jsNative.kt

```diff
@@ -19,4 +19,4 @@ package androidx.compose.foundation.text
 import androidx.compose.ui.input.key.KeyEvent
 
 actual val KeyEvent.isTypedEvent: Boolean
-    get() = TODO("implement native KeyEvent.isTypedEvent")
+    get() = false // on UIKit physical keyboard not handled as text input
```

Comment:
Can KeyEvent come from a screen-keyboard? Is this property only for KeyEvents from physical keyboard? 

Maybe this file needs to be ranamed: TextFieldKeyInput.jsNative.kt -> TextFieldKeyInput.uikit.kt

Reply:
1) This property only for KeyEvents from physical keyboard
2) Thanks, I will rename to uikit.kt

Reply:
Done